src/components/Items/ProfileContact.js

```diff
@@ -3,7 +3,7 @@ import Mi from 'react-native-vector-icons/MaterialIcons';
 import PropTypes from 'prop-types';
 import {Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
-import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
+import IconChecklist from '../../assets/icons/Ic_Checklist';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `IconChecklist` has been updated correctly. However, please ensure that the renaming of `MemoIc_Checklist` to `IconChecklist` is reflected throughout the entire codebase to maintain consistency.
```diff
- import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
+ import IconChecklist from '../../assets/icons/Ic_Checklist';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->